README.md

```diff
@@ -109,6 +109,12 @@ or if you're using with babel-loader, you can
 }
 ```
 
+**If you want to use aria attributes in your SVGs**, set this SVGO plugin option:
+
+```js
+{ removeUnknownsAndDefaults: false }
```

Comment:
Would it make sense to add it by default even if the option is not passed or disabled?

https://github.com/boopathi/react-svg-loader/blob/master/src/svgo.js#L6